ui/src/app/model/migrate.ts

```diff
@@ -38,4 +38,18 @@ export interface ISourceAndTargetDetails {
     SpannerDatabaseUrl: string
     SourceDatabaseName: string
     SourceDatabaseType: string
+}
+
+export interface ITables {
+    TableList: string[]
+}
+
+export interface ITableState {
+    TableName: string
+    TableId: string
+    isDeleted: boolean
+}
+
+export interface ITableStateList {
```

Comment:
`ITableList` is not a linear object and incapsulates UI specific information (if the table is currently marked delete or not) along with tableName and Id.

Reply:
I meant ITableStateList , looks like you have removed it

Reply:
It got deleted due to an issue in rebasing. I understood the point though, `ITables` is used in frontend backend interaction where `ITableStateList` was redundant since `ITableState[]` would suffice here. Have removed it now.